### ISSUE TITLE
BallView Crashes Upon Loading File

### ISSUE THREAD

**Issue by anhi** [BALL-Project/ball#235]
**Reported by jepson c on 20 Mar 39558785 00:26 UTC**
Version is 1.3 Beta 2.

I created a new project, worked for a while, saved it and then closed it and tried to reopen it only to find that this cause the program to crash.  I had the same problem with 1.3 beta 1 and it doesn't seem to have been corrected (not sure if any attempt was made?).

File can be downloaded here:

http://www.mediafire.com/?sharekey=c5dd05f2095b1e77c2b435915e8821d7e04e75f6e8ebb871

**Comment by anhi** [comment]
**Commented by anhi on 23 Jan 39560546 14:13 UTC**
Thanks! I was convinced that we had squashed this bug. I'll look into it!

**Comment by anhi** [comment]
**Commented by dstoeckel on 30 Dec 39571679 05:46 UTC**
This should really be fixed before 1.3 gets released. Setting the priority to blocker.

**Comment by anhi** [comment]
**Commented by anhi on 6 Sep 39581290 17:20 UTC**
Ok, I hope I finally got rid of that one... This was a pretty nasty combination of

 1. a large array on the stack that makes Windows choke (stack limit of 1Mb)
 2. a 64/32 bit inconsistency
 3. Windows losing a binary flag for a file and replacing \r, \eof, ... in the binary XDR output...

I've commited a fix and will prepare a new Windows installer in the next few days.
